src/scan.js

```diff
@@ -1,8 +1,20 @@
 const path = require('path');
 const fs = require('fs');
 
-module.exports = function (pathName, dts) {
+// ./Button-Next => ./ButtonNext
+const componentNameFormat = (str) => {
+  return str.split('-').map(subStr => upperCaseFirstChar(subStr)).join('')
+}
+
+// ./button-next => ./Button-next
```

Comment:
Can you explain a little bit about what is the expected output? I mean the actual result we aim to achieve, not about the implementation...